jbpm-flow/src/main/java/org/jbpm/workflow/instance/impl/WorkflowProcessInstanceImpl.java

```diff
@@ -79,39 +79,47 @@
 	@Deprecated // this should be deleted in 7.0.x
 	private boolean deprecatedIdStrategy = Boolean.getBoolean("jbpm.v5.id.strategy");
 	private AtomicLong singleNodeInstanceCounter = new AtomicLong(0);
-	
+
 	private Map<String, List<EventListener>> eventListeners = new HashMap<String, List<EventListener>>();
 	private Map<String, List<EventListener>> externalEventListeners = new HashMap<String, List<EventListener>>();
+
 	private List<String> completedNodeIds = new ArrayList<String>();
 	private List<String> activatingNodeIds;
 	private Map<String, Integer> iterationLevels = new HashMap<String, Integer>();
 	private int currentLevel;
 	private boolean persisted = false;
 	private Object faultData;
-	
+
 	private boolean signalCompletion = true;
 
+	public static final EventListener EMPTY_EVENT_LISTENER = new EventListener() {
```

Comment:
could we externalize this class so we could avoid import of WorkflowProcessInstanceImpl in here: https://github.com/droolsjbpm/jbpm/pull/388/files?w=1#diff-625e8789a66222e0e5be169c3f8c5af9R19